frontend/app/component/form/polling_station/PollingStationForm.test.tsx

```diff
@@ -7,29 +7,48 @@ import { ErrorResponse, PollingStation } from "@kiesraad/api";
 
 import { PollingStationForm } from "./PollingStationForm";
 
+function getInputs() {
+  const result = {
+    number: screen.getByRole("textbox", { name: "Nummer" }),
+    name: screen.getByRole("textbox", { name: "Naam" }),
+    numberOfVoters: screen.getByRole("textbox", { name: "Aantal kiesgerechtigden Optioneel" }),
+    street: screen.getByRole("textbox", { name: "Straatnaam" }),
+    houseNumber: screen.getByRole("textbox", { name: "Huisnummer" }),
+    houseNumberAddition: screen.getByRole("textbox", { name: "Toevoeging" }),
+    postalCode: screen.getByRole("textbox", { name: "Postcode" }),
+    locality: screen.getByRole("textbox", { name: "Plaats" }),
+    typeOptionFixedLocation: screen.getByRole("radio", { name: "Vaste locatie" }),
+    typeOptionSpecial: screen.getByRole("radio", { name: "Bijzonder" }),
+    typeOptionMobile: screen.getByRole("radio", { name: "Mobiel" }),
+  };
+
+  return result;
+}
+
 async function fillForm(user: UserEvent, testPollingStation: PollingStation | Omit<PollingStation, "id">) {
-  await user.type(await screen.findByRole("textbox", { name: "Nummer" }), testPollingStation.number.toString());
-  await user.type(await screen.findByRole("textbox", { name: "Naam" }), testPollingStation.name.toString());
-  await user.type(await screen.findByRole("textbox", { name: "Straatnaam" }), testPollingStation.street.toString());
-  await user.type(
-    await screen.findByRole("textbox", { name: "Huisnummer" }),
-    testPollingStation.house_number.toString(),
-  );
+  const inputs = getInputs();
+  await user.type(inputs.number, testPollingStation.number.toString());
+  await user.type(inputs.name, testPollingStation.name.toString());
+  await user.type(inputs.street, testPollingStation.street.toString());
+  await user.type(inputs.houseNumber, testPollingStation.house_number.toString());
   if (testPollingStation.house_number_addition) {
-    await user.type(
-      await screen.findByRole("textbox", { name: "Toevoeging" }),
-      testPollingStation.house_number_addition.toString(),
-    );
+    await user.type(inputs.houseNumberAddition, testPollingStation.house_number_addition.toString());
+  }
+  await user.type(inputs.postalCode, testPollingStation.postal_code.toString());
+  await user.type(inputs.locality, testPollingStation.locality.toString());
+  await user.type(inputs.numberOfVoters, String(testPollingStation.number_of_voters?.toString()));
+
+  switch (testPollingStation.polling_station_type) {
+    case "FixedLocation":
+      await userEvent.click(inputs.typeOptionFixedLocation);
+      break;
+    case "Special":
+      await userEvent.click(inputs.typeOptionSpecial);
+      break;
+    case "Mobile":
+      await userEvent.click(inputs.typeOptionMobile);
```

Comment:
What's the reason for using `userEvent` here instead of `user`?

Reply:
changed to user